src/sections/HomePartners/index.css

```diff
@@ -0,0 +1,42 @@
+@import "../../styles/layout.css";
+@import "../../styles/breakpoints.css";
+
+.title {
+  @include Breakpoint-desktopOnly {
+    max-width: $four-columns;
+  }
+
+  @include Breakpoint-mobileOnly {
+    margin-right: auto;
+  }
+}
+
+.quotes {
+  margin-top: $spacing-28;
+  margin-left: calc($three-columns - $gutter);
+
+  @include Breakpoint-mobileOnly {
+    margin-top: $spacing-56;
+    margin-left: 0;
+  }
+}
+
+.about {
```

Comment:
Fixed height to 5 lines, like Inês said.

Reply:
👍